### DISCUSSION TITLE
Request for Guidance on Creating an SRT File

### DISCUSSION THREAD

**Discussion by ageutube6** [deepgram/community#170] [General help]
I am currently working on a project that requires the creation of SRT (SubRip Subtitle) files. Despite my best efforts, I am finding it a bit challenging to navigate the process.

I was wondering if you could provide me with some detailed guidance or resources on how to generate these files. Your expertise would be greatly appreciated, and I am sure that your advice would be very helpful in overcoming the challenges I am currently facing.

**Comment by jjmaldonis** [comment]
Hey @ageutube6! Happy to help. Check out [this blog post](https://blog.deepgram.com/generate-webvtt-srt-captions-nodejs/) to start. It uses Deepgram's Node SDK to create an SRT file using a prerecorded audio file. Our Python SDK has similar functionality to create an SRT file from an API response.

I can write up some more info if it's helpful as well. If that'd be useful, what programming language are you using? And are you using a re-recorded audio file, or is the audio being streamed? Are there are specifics for your use case?
